src/test/java/core/basesyntax/service/FileWriterTest.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.io.File;
+import java.util.List;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static final String testTransactionFilePath = "src/main/resources/transaction.csv";
+    private static final String dayReportTestFilePath = "src/main/resources/dayreport.csv";
+
+    @Test
+    public void fileWriter_isReportFileExist_OK() {
```

Comment:
you should check the content of written file too

Reply:
Done